### PR TITLE
fix jinja context in salt 2019.2

### PR DESCRIPTION
**Pull request by kiniou**
### PR progress checklist (to be filled in by reviewers)

- [x] Changes to documentation are appropriate (or tick if not required)
- [x] Changes to tests are appropriate (or tick if not required)
- [x] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [ ] `[feat]`     A new feature
- [x] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [ ] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [ ] `[docs]`     Documentation changes
- [ ] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Related issues and/or pull requests

No issues that i'm aware of.

### Describe the changes you're proposing

This is just a fix following recent breaking changes in jinja for saltstack 2019.2+

### Pillar / config required to test the proposed changes

No changes in pillar, just fixing jinja rendering error.

### Documentation checklist

- [ ] Updated the `README` (e.g. `Available states`).
- [ ] Updated `pillar.example`.

### Testing checklist

- [ ] Included in Kitchen (i.e. under `state_top`).
- [ ] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [ ] Updated the relevant test pillar.

### CODE REVIEW

File: debootstrap/state.jinja [hunk 1]
```diff
@@ -119,7 +119,7 @@ debootstrap_{{ data.id }}_sources_list:
     - mode: 644
     - template: jinja
     - context:
-        data: {{ data }}
+        data: {{ data|tojson }}
```
Comment (myii) [inline debootstrap/state.jinja]:
```suggestion
        data: {{ data|json }}
```

For more info, please refer back to this comment: https://github.com/saltstack-formulas/consul-formula/pull/40#issuecomment-472728517.

**Review by myii — CHANGES_REQUESTED** [review]
@kiniou Thanks for the PR.  Please see the inline feedback.

**Review by myii — APPROVED** [review]

File: debootstrap/state.jinja (hunk 1 above)
Comment (kiniou) [reply debootstrap/state.jinja]: 👍 I was not aware of the json and mostly followed https://docs.saltstack.com/en/latest/topics/releases/2019.2.1.html

Thanks for the tip 😉

**Comment by myii** [comment]
Merged, thanks @kiniou!

**Pull request merged**